tools/SWTbot-Testing/src/swtbot_testing/MavenSwtest.java

```diff
@@ -0,0 +1,453 @@
+            bot.button( "Next >" ).click();
+
+            bot.textWithLabel( "Liferay Tomcat directory" ).setText(
+                "D:\\6.2\\liferay-portal-tomcat-6.2-ce-ga6-20160112152609836\\liferay-portal-6.2-ce-ga6" );
+
+            bot.button( "Next >" ).click();
+            bot.button( "Finish" ).click();
+            assertEquals( "", bot.textWithLabel( "New profile id:" ).getText() );
+            bot.comboBoxWithLabel( "Liferay version:" ).setSelection( "6.2.5" );
+
+            bot.button( "OK" ).click();
+            bot.textWithLabel( "Active profiles:" ).setText( "Liferay-v6.2-CE-(Tomcat-7)" );
+
+            bot.toolbarButtonWithTooltip( "Select Active Profiles..." ).click();
+            bot.button( "OK" ).click();
+            bot.button( "Next >" ).click();
+            /* �鿴next page */
+            bot.radio( "Liferay MVC" ).isSelected();
+            assertTrue( bot.radio( "Liferay MVC" ).isSelected() );
+            assertFalse( bot.radio( "JSF 2.x" ).isSelected() );
+            assertFalse( bot.radio( "Spring MVC" ).isSelected() );
+            assertFalse( bot.radio( "Vaadin" ).isSelected() );
+
+            assertEquals( "", bot.textWithLabel( "Portlet name:" ).getText() );
+            assertEquals( "", bot.textWithLabel( "Display name:" ).getText() );
+            assertEquals( "", bot.textWithLabel( "Archetype:" ).getText() );
+
+            assertTrue( bot.button( "< Back" ).isEnabled() );
+            assertFalse( bot.button( "Next >" ).isEnabled() );
+            assertTrue( bot.button( "Finish" ).isEnabled() );
+            assertTrue( bot.button( "Cancel" ).isEnabled() );
+            bot.button( "Finish" ).click();
+            bot.sleep( 2000 );
+            // finish֮��check��ɵ�java�ļ�
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).expand();
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "pom.xml" ).select();
+            assertTrue( bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "pom.xml" ).isVisible() );
+            bot.sleep( 2000 );
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).expand();
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "src" ).expand();
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "src" ).getNode( "main" ).expand();
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "src" ).getNode( "main" ).getNode(
+                "webapp" ).expand();
+            bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "src" ).getNode( "main" ).getNode(
+                "webapp" ).getNode( "view.jsp" ).select();
+            assertTrue(
+                bot.tree().getTreeItem( "CreateMavenProjectTesting" ).getNode( "src" ).getNode( "main" ).getNode(
+                    "webapp" ).getNode( "view.jsp" ).isVisible() );
+        }
+        else
```

Comment:
leave the common tests and put the difference tests in if{}else{}, reduce repeating codes